DiseasesExpanded/Cures/HungermsVaccineConfig.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace DiseasesExpanded
 {
-    class HungermsVaccineConfig : IEntityConfig
+    class HungermsVaccineConfig// : IEntityConfig
```

Comment:
restore IEntityConfig

Reply:
Why? The entity isn't used but appeares here:
![image](https://github.com/pether-pg/ONI_Mods_byPether/assets/74019811/a1cf729f-4d7d-4b48-8160-0842ffa6473b)

Reply:
each entity config must use IEntityConfig. please restore it and never remove again

Reply:
ok, but what about commenting out the whole class like u did with obsolete rooms in Rooms Expanded mod?

Reply:
I see. Hungerms Vaccine is integrated in other mods like PrintingPodRecharge. Why is the recipe commented out?